matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/timeline/DefaultTimeline.kt

```diff
@@ -775,8 +784,7 @@ internal class DefaultTimeline(
             }
             if (!filterEdits) return@filter false
 
-            val filterRedacted = !settings.filters.filterRedacted || it.root.isRedacted()
-
+            val filterRedacted = !settings.filters.filterRedacted || it.root.isRedacted().not()
```

Comment:
For clarity and coherency (I had to read it twice) we should write:

```
val filterRedacted = settings.filters.filterRedacted && it.root.isRedacted()

return !filterRedacted
```

Reply:
Yup totally, my brain was off :O